plugins/scsi/fu-scsi-device.c

```diff
@@ -346,7 +347,7 @@ fu_scsi_device_write_firmware(FuDevice *device,
 			      GError **error)
 {
 	FuScsiDevice *self = FU_SCSI_DEVICE(device);
-	guint32 chunksz = 0x1000;
+	guint32 chunksz = self->write_buffer_size ?: 0x1000;
```

Comment:
rather than doing this, could we set `self->write_buffer_size = 0x1000` in `_init()`?